tensorflow/core/api_def/base_api/api_def_Acos.pbtxt

```diff
@@ -2,3 +2,4 @@ op {
   graph_op_name: "Acos"
   summary: "Computes acos of x element-wise."
 }
+
```

Comment:
This extra line should not be here, this file should not be touched at all.

I suggest `git checkout HEAD -- tensorflow/core/api_def/base_api/api_def_Acos.pbtxt`